docker/melodic/Dockerfile

```diff
@@ -11,8 +10,8 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 RUN mkdir -p /root/catkin_ws/src
 WORKDIR /root/catkin_ws/src
 RUN /bin/bash -c '. /opt/ros/melodic/setup.bash; catkin_init_workspace'
-RUN git clone https://github.com/SMRT-AIST/fast_gicp.git --recursive
-RUN git clone https://github.com/koide3/hdl_graph_slam.git
+RUN git clone https://github.com/SMRT-AIST/fast_gicp.git --recursive --depth=1
```

Comment:
I added the `--depth=1` option to avoid `git clone` errors.

https://github.com/KeisukeShima/ndt_omp/runs/4639280919?check_suite_focus=true#step:4:3179